infra/feast-operator/api/v1alpha1/featurestore_types.go

```diff
@@ -20,24 +20,59 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+const (
+	// Feast phases:
+	ReadyPhase   = "Ready"
+	PendingPhase = "Pending"
+	FailedPhase  = "Failed"
+
+	// Feast condition types:
+	ClientReadyType   = "Client"
+	RegistryReadyType = "Registry"
+	ReadyType         = "FeatureStore"
+
+	// Feast condition reasons:
+	ReadyReason          = "Ready"
+	FailedReason         = "FeatureStoreFailed"
+	RegistryFailedReason = "RegistryDeploymentFailed"
+	ClientFailedReason   = "ClientDeploymentFailed"
+
+	// Feast condition messages:
+	ReadyMessage         = "FeatureStore installation complete"
```

Comment:
maybe change to `Feast installation complete`

Reply:
done

Reply:
well, if the kind is `FeatureStore`, it was not that wrong. Anyway, it's fine. Can I merge it @tchughesiv?

Reply:
i agree actually... made more sense the other way. I've reverted that last commit... and yes, it can be merged!